LaTeX/LaTeX.sublime-syntax

```diff
@@ -225,6 +225,8 @@ contexts:
         - include: general-commands
         - include: macro-braces
 
+    - include: array-preamble
```

Comment:
Thank you, I didn't fully understand the structure of the file before. I will update the PR.